packages/odyssey-storybook/package.json

```diff
@@ -51,27 +51,30 @@
     "@types/react": "^18.2.55",
     "@types/react-dom": "^18.2.19",
     "@vitejs/plugin-react-swc": "^3.6.0",
-    "axe-core": "^4.8.4",
+    "axe-core": "^4.9.1",
     "axe-playwright": "^2.0.1",
     "babel-loader": "^9.1.3",
+    "concurrently": "^8.2.2",
+    "http-server": "^14.1.1",
     "markdown-to-jsx": "^7.4.1",
-    "npm-run-all": "^4.1.5",
-    "playwright": "^1.41.2",
+    "playwright": "^1.43.1",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "storybook": "^7.6.16",
     "storybook-addon-rtl-direction": "^0.0.19",
     "typescript": "^5.3.3",
-    "vite": "^5.1.3"
+    "vite": "^5.1.3",
+    "wait-on": "^7.2.0"
   },
   "scripts": {
     "build": "storybook build --output-dir dist",
+    "coverage": "yarn build && yarn concurrently --kill-others --success first --names \"STORYBOOK-SERVER,INTERACTION-TEST\" --prefix-colors \"magenta,green\" \"http-server storybook-static --port 6006 --silent\" \"wait-on tcp:127.0.0.1:6006 && yarn test-storybook --url http://127.0.0.1:6006\" --coverage",
+    "dev:interactionTest": "yarn build && yarn concurrently --kill-others --success first --names \"STORYBOOK-SERVER,INTERACTION-TEST\" --prefix-colors \"magenta,green\" \"http-server storybook-static --port 6006 --silent\" \"wait-on tcp:127.0.0.1:6006 &&  yarn test-storybook --url http://127.0.0.1:6006\"",
     "prepack": "yarn build && yarn exec prepack",
     "start:storybook": "storybook dev -p 6006",
     "start:types": "tsc --build --watch",
-    "start": "npm-run-all --parallel start:types start:storybook",
-    "test:storybook": "test-storybook --maxWorkers 4",
+    "start": "yarn concurrently npm:start:types npm:start:storybook",
```

Comment:
Do you need the `npm:` prefixes here? I believe they're short form for `npm run` and we didn't need that previously?

Reply:
With `concurrently`, it needs the `npm:` prefix when you're defining a script name from `package.json`. `npm-run-all` only worked with script names; not custom commands.